examples/opacus/README.md

```diff
@@ -1,28 +1,60 @@
-# Differentially Private Federated Learning using Opacus, PyTorch and Flower
+# Training with Sample-Level Differential Privacy using Opacus Privacy Engine
 
-This example contains code demonstrating how to include the Opacus library for training a model using DP-SGD. The code is adapted from multiple other examples:
+In this example, we demonstrate how to train a model with differential privacy (DP) using Flower. We employ PyTorch and integrate the Opacus Privacy Engine to achieve sample-level differential privacy. This setup ensures robust privacy guarantees during the client training phase. The code is adapted from the [PyTorch Quickstart example](https://github.com/adap/flower/tree/main/examples/quickstart-pytorch).
 
-- PyTorch Quickstart
-- Simulation Quickstart
-- Simulation Extended Example
+For more information about DP in Flower please refer to the [tutorial](https://flower.ai/docs/framework/how-to-use-differential-privacy.html). For additional information about Opacus, visit the official \[website\] (https://opacus.ai/).
 
-## Requirements
+## Environments Setup
 
-- **Flower** nightly release (or development version from `main` branch) for the simulation, otherwise normal Flower for the client
-- **PyTorch** 1.7.1 (but most likely will work with older versions)
-- **Ray** 1.4.1 (just for the simulation)
-- **Opacus** 0.14.0
+Start by cloning the example. We prepared a single-line command that you can copy into your shell which will checkout the example for you:
 
-## Privacy Parameters
+```shell
+git clone --depth=1 https://github.com/adap/flower.git && mv flower/examples/sample-level-dp-opacus . && rm -rf flower && cd sample-level-dp-opacus
```

Comment:
This will only work if we rename the directory of this example as `sample-level-dp-opacus`